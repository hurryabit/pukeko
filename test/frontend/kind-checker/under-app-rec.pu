data Typ a b = Con (Typ a)
