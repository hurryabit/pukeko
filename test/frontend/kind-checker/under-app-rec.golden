test/frontend/kind-checker/under-app-rec.pu:1:16: cannot unify kinds _3 -> * and *
